esmvaltool/cmorizers/obs/download_scripts/download_cds_satellite_soil_moisture_monthly.py

```diff
@@ -0,0 +1,159 @@
+"""
+Download cds-satellite-soil-moisture.
+
+Script to download cds-satellite-soil-moisture
+from the Climate Data Store(CDS).
+"""
+
+import argparse
+import os
+import subprocess
+
+import yaml
+
+import cdsapi
+
+
+def download_cds_satellite_soil_moisture():
+    """Download dataset cds-satellite-albedo."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--config_file',
+                        '-c',
+                        default=os.path.join(os.path.dirname(__file__),
+                                             'config-user.yml'),
+                        help='Config file')
+    parser.add_argument('--frequency',
+                        '-f',
+                        default=None,
```

Comment:
```suggestion
                        default=None,
                        choices=['daily', 'monthly']
```
Although the script name says `_monthly'.